packages/apollo-link-http/README.md

```diff
@@ -1,54 +1,49 @@
 ---
-title: Http Link
+title: apollo-link-http
```

Comment:
Are you going to change all the other links to have their package name as the title? Right now, it looks inconsistent on the deploy preview.

Reply:
Yeah, just wondering if you think it's a good idea!

Reply:
I like it better with the package name :)

Reply:
Dope. will do the rest